src/router/index.ts

```diff
@@ -87,13 +87,19 @@ export class SpaceRouter {
         });
 
         SpaceRouter.router.afterEach((to) => {
-            if (config.get('GTAG_ID') !== 'DISABLED') GTag.setPageView(to);
-            if (SpaceRouter.router.app.$store.state.error.visibleAuthorizationError) { SpaceRouter.router.app.$store.commit('error/setVisibleAuthorizationError', false); }
-            const isDomainOwner = SpaceRouter.router.app.$store.getters['user/isDomainOwner'];
+            // set GTag
+            const gtagId = config.get('GTAG_ID');
+            if (!gtagId || gtagId !== 'DISABLED') GTag.setPageView(to);
```

Comment:
```suggestion
            if (gtagId || gtagId !== 'DISABLED') GTag.setPageView(to);
```
I think there should be `gtagId` to `setPageview()`

Reply:
Oh, it's correct.
`GTag.setPageView()` function's parameter is the route to go.